wasm/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "tsl",
+  "version": "0.0.1",
```

Comment:
Maybe we can keep track here of the  current TSL version which is available here: https://github.com/ovh/tsl/blob/master/cmd/version.go ?

Reply:
Done in #3679244